### PR TITLE
The --config argument now takes a file rather than a directory

### PR DESCRIPTION
**Pull request by mjs**
Now that there's only one configuration file it doesn't need to take a directory.

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ Create a new database called 'cacophonytest'
   - `\q`
 
 Migrate the database to the latest version
```
Comment (clare) [inline test/README.md]: Actually maybe we should name the test config file testapp.js.   That would make sure that we are getting the right file.

File: config.js [hunk 2]
```diff
@@ -1,45 +1,38 @@
...
-    for (var key in config) {
-        exports[key] = config[key];
-    }
+  checkDatabaseConfigAvailable(config);
 
-    testDatabaseConfigAvailable(config);
-
-    return exports;
+  return exports;
 }
```
Comment (clare) [inline config.js]: Might be worth using path.resolve(configDirPath) to get the full path here for this error.   That way user can see exactly where the program is looking.

Comment (mjs) [reply config.js]: Already done. The first thing that loadConfig() does is path.resolve(configPath).

File: test/README.md (hunk 1 above)
Comment (mjs) [reply test/README.md]: Yep, that's a good idea. Eventually I think we want the generation of the test config file to be automated but using a more obvious configuration file name now is a good idea.

I'll add that to this PR.

File: config.js (hunk 2 above)
Comment (clare) [reply config.js]: Sorry missed that.

**Pull request merged**